lib/tasks/hyacinth/setup.rake

```diff
@@ -56,5 +37,107 @@ namespace :hyacinth do
         end
       end
     end
+
+    desc "Add Descriptive Metadata category and Title dynamic fields"
+    task seed_dynamic_field_entries: :environment do
+      user = User.find_by(email: default_user_accounts[0][:email]);
+      if !user
+        puts Rainbow("seed_dynamic_field_entries requires default Admin user. Run default_users task first").red.bright
+        next
+      end
+      df_category = DynamicFieldCategory.find_by(display_label: "Descriptive Metadata")
+      if df_category
+        puts Rainbow("dynamic field category 'Descriptive Metadata' already exists (skipping).").blue.bright
+      else
+        df_category = DynamicFieldCategory.create!(display_label: "Descriptive Metadata")
+        puts Rainbow("Created dynamic field category 'Descriptive Metadata'").green
+      end
+
+      df_group = DynamicFieldGroup.find_by(display_label: "Title")
+      if df_group
+        puts Rainbow("dynamic field group 'Title' already exists (skipping).").blue.bright
+      else
+        df_group = DynamicFieldGroup.create!(
+          string_key: 'title',
+          parent: df_category,
+          display_label: 'Title',
+          created_by: user,
+          updated_by: user,
+          parent_type: "DynamicFieldCategory",
+        )
+        puts Rainbow("Created dynamic field group 'Title'").green
+      end
+
+
+      default_dynamic_fields = [
+        {
+          display_label: 'Sort Portion',
+          string_key: 'title_sort_portion',
+          dynamic_field_group: df_group,
+          field_type: 'string',
+          created_by: user,
+          updated_by: user,
+          filter_label: '',
+          controlled_vocabulary: '',
+          select_options: '{}',
+          additional_data_json: '{}'          
+        },
+        {
+          display_label: 'Non-Sort Portion',
+          string_key: 'title_non_sort_portion',
+          dynamic_field_group: df_group,
+          field_type: 'string',
+          created_by: user,
+          updated_by: user,
+          filter_label: '',
+          controlled_vocabulary: '',
+          select_options: '{}',
+          additional_data_json: '{}'          
+        }
+      ]
+
+      default_dynamic_fields.each do |field|
+        if DynamicField.exists?(string_key: field[:string_key])
+          puts Rainbow("dynamic field #{field[:display_label]} already exists (skipping).").blue.bright
+        else
+          DynamicField.create!(
```

Comment:
Here, you should be able to do `DynamicField.create!(**field)` unless there's some reason to be selective about what variables to use. You may not even need the two asterisk.